### PR TITLE
Website/updatefooter

### PR DESCRIPTION
**Pull request by lilibobear**

### CODE REVIEW

File: www/docusaurus.config.js [hunk 1]
```diff
@@ -137,6 +137,13 @@ const config = {
         ],
       },
       footer: {
+        logo: {
+          alt: "Firezone Logo",
+          src: "img/logo.svg",
+          href: 'https://www.firezone.dev',
```
Comment (jamilbk) [inline www/docusaurus.config.js]: Consistent quote style:

```suggestion
          href: "https://www.firezone.dev",
```

File: www/docusaurus.config.js [hunk 2]
```diff
@@ -137,6 +137,13 @@ const config = {
...
+          width: 160,
+          height:50,
```
Comment (jamilbk) [inline www/docusaurus.config.js]: Consistent spacing:

```suggestion
          height: 50,
```

File: www/docusaurus.config.js [hunk 3]
```diff
@@ -171,10 +195,14 @@ const config = {
...
               },
+              {
+                label: "LinkedIn",
+                href: "https://www.linkedin.com/company/firezonehq/",
+              },
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} Firezone, Inc.`,
+        copyright: `Copyright © ${new Date().getFullYear()} Firezone, Inc. Wireguard® is a registered trademark of Jason A. Donenfeld.`,
```
Comment (jamilbk) [inline www/docusaurus.config.js]: Properly spell WireGuard

```suggestion
        copyright: `Copyright © ${new Date().getFullYear()} Firezone, Inc. WireGuard® is a registered trademark of Jason A. Donenfeld.`,
```

**Review by jamilbk — CHANGES_REQUESTED** [review]
Made some minor suggestions

**Review by lilibobear — COMMENTED** [review]
Incorporated comments.

**Review by jamilbk — APPROVED** [review]

**Pull request merged**